Change notes for edge-23.11.2 (#11600)
linkerd/linkerd2: CHANGES.md

# Changes

## edge-23.11.2

This edge release contains observability improvements and bug fixes to the
Destination controller, and a refinement to the multicluster gateway resolution
logic.

* Fixed an issue where the Destination controller could stop processing service
  profile updates, if a proxy subscribed to those updates stops reading them;

>   this is a followup to the issue [#11491] fixed in edge-23.10.3 ([#11546])

Reviewer: tiny nit: should we maybe make the reference to `edge-23.10.3` a link?
```suggestion
  this is a followup to the issue [#11491] fixed in [edge-23.10.3] ([#11546])
```

* In the Destination controller, added informer lag histogram metrics to track

>   whenever the objects tracked are falling behind the state in the

Reviewer: nit: maybe:
```suggestion
  whenever the Kubernetes objects watched by the controller are falling behind the state in the
```

  kube-apiserver ([#11534])
* In the multicluster service mirror, extended the target gateway resolution
  logic to take into account all the possible IPs a hostname might resolve to,

>   not just the first one (thanks @MrFreezeex!) ([#11499])

Reviewer: nit: maybe
```suggestion
  rather than just the first one (thanks @MrFreezeex!) ([#11499])
```
